client/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import Home from "./components/Home3rd";
+import Cities from "./components/Cities";
+// import Error from "./components/Error";
+import Login from "./components/Header/Login";
+import SignUp from "./components/Header/SignUp";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import Itinerary from "./components/Itinerary";
+import Activity from "./components/Activity";
+
+class App extends Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <div>
+          <Header />
+          <Route exact path="/" component={Home} />
+          <Route path="/cities" component={Cities} />
+          <Route path="/login" component={Login} />
+          <Route path="/signup" component={SignUp} />
+          <Route path="/itineraries/:city" component={Itinerary} />
+          <Route path="/itineraries/:id" component={Activity} />
+          <Footer />
+          {/* <Route component = {Error}/> */}
+        </div>
+      </BrowserRouter>
+    );
+  }
+}
+
+export default App;
+
+// state = {
```

Comment:
I recommend not committing any commented out code usually since it may lead to confusion in the future.